src/main/java/org/gitlab4j/api/models/MergeRequest.java

```diff
@@ -273,6 +277,7 @@ public void setSquashCommitSha(String squashCommitSha) {
         this.squashCommitSha = squashCommitSha;
     }
 
+    @Deprecated
```

Comment:
Same here, indicate what is the replacement:

```suggestion
    /**
     * @deprecated since 15.6, use {@link #getDetailedMergeStatus()} instead.
     */
    @Deprecated
```